SourceCode/frontend/src/Modules/Events/EventContainer.js

```diff
@@ -24,27 +28,23 @@ const EventContainer = (props) => {
         }
     }, []);
 
-    const handleSubmit = (data) => {
-        if(data) {
+    const handleSubmit = async (query) => {
+        if(query) {
+            setLoading(true);
             let foodBankUri = API_URL.FOODBANK_LIST;
-            let foodbankResponse = {
-                "foodbanks": [
-                    {
-                        "address": "3960 Brookham DR",
-                        "city": "Grove City",
-                        "state": "OH",
-                        "zip": "43123",
-                        "name": "Mid-Ohio Foodbank",
-                        "nickname": "Mid-Ohio Foodbank",
-                        "display_url": "https://www.midohiofoodbank.org/get-help/get-food/",
-                        "fb_agency_locator_url": "https://www.midohiofoodbank.org/get-help/get-food/",
-                        "fb_url": "https://www.midohiofoodbank.org/",
-                        "fb_fano_url": "https://www.feedingamerica.org/find-your-local-foodbank/mid-ohio-foodbank"
-                    }
-                ]
-            };
-            setFoodBankData(foodbankResponse);
-            setFoodBankResponse(true);
+            // Going to use axios for now
```

Comment:
I had a better experience with using axios directly instead of the abstracted `ajaxGet`. Can come back to it later.